src/common/low_precision_transformations/src/concat.cpp

```diff
@@ -111,9 +117,21 @@ bool ConcatTransformation::transform(TransformationContext& context, ngraph::pat
 
         if (!allDequantizationShiftAreZero) {
             auto subtractInput = dequantization.subtract == nullptr ?
-                std::make_shared<ngraph::opset1::Constant>(deqPrecision, targetShape, std::vector<float>({ 0.f })) :
+                (allDequantizationShiftConvertAreNotZero ?
+                    std::make_shared<ngraph::opset1::Constant>(
+                        element::u8,
+                        targetShape,
+                        std::vector<unsigned int>({ 0u })) :
```

Comment:
1. `element::u8`: use original precision from the model, please.
2. minor: `std::vector<unsigned int>({ 0u }))`: use more appropriate general type instead `unsigned int`. I think `float` should be used here.

Reply:
PrecisionBeforeConvert is now used to save the original precision of the model. Thank you